tools/dev.js

```diff
@@ -0,0 +1,87 @@
+const path = require("path");
+const { createServer } = require("http");
+const chokidar = require("chokidar");
+const karma = require("karma");
+const serve = require("serve-handler");
+const colors = require("colors");
+const serveConfig = require("../serve.json");
+const { Builder } = require("./builder");
+
+const KARMA_PORT = 9876;
+const SERVE_PORT = 5000;
+
+const karmaConfig = karma.config.parseConfig(
+  path.join(__dirname, "../karma.conf.js"),
+  {
+    browsers: ["ChromeHeadless"],
```

Comment:
But...but...👀

Reply:
This will be cli arg. Using this during prototyping as it doesn't open a new window

Reply:
Yes, but...but... why not Firefox! 😂